Sources/OsmEdit/OAEditPOIData.mm

```diff
@@ -62,6 +63,85 @@ -(void)initTags:(OAEntity *) entity
     return _allTranslatedSubTypes;
 }
 
+-(NSArray<NSString *>*)getAllTags {
+    NSMutableSet <NSString *>* stringSet = [[NSMutableSet alloc] init];
+    NSMutableSet <NSString *>* value = [[NSMutableSet alloc] init];
+    NSMutableArray *result = [[NSMutableArray alloc] init];
+
+    for (OAPOIType* poi in [_allTranslatedSubTypes allValues])
+    {
+        [self addPoiToStringSet:poi stringSet:stringSet values:value];
+    }
+
+    for (NSString* string in stringSet)
+        [result addObject: string];
+
+    for (NSString* string in value)
+        [result addObject: string];
```

Comment:
Why values added to tags? It should be two autocomplete toolbars depending on tag/value text field selected.